SmartDeviceLink/SDLIAPSession.m

```diff
@@ -24,13 +31,23 @@ @implementation SDLIAPSession
 - (instancetype)initWithAccessory:(EAAccessory *)accessory forProtocol:(NSString *)protocol {
     NSString *logMessage = [NSString stringWithFormat:@"SDLIAPSession initWithAccessory:%@ forProtocol:%@", accessory, protocol];
     [SDLDebugTool logInfo:logMessage];
+    
+    if ([protocol isEqualToString:@"com.smartdevicelink.prot0"]){
```

Comment:
This constant is defined in `SDLIAPTransport.m`, it should probably be shared, but adding it to the `SDLIAPTransport.h` header will force this to be a minor version change.